src/L2YnOFTAdapterUpgradeable.sol

```diff
@@ -25,8 +23,7 @@ contract L2YnOFTAdapterUpgradeable is OFTAdapterUpgradeable, RateLimiter {
      */
     function initialize(address _owner) external virtual initializer {
         __OFTAdapter_init(_owner);
-        __Ownable_init();
-        _transferOwnership(_owner);
```

Comment:
why does `_transferOwnership` dissapears?

Reply:
`__Ownable_init(_owner)` sets the owner now and we no longer need the `_transferOwnership` call.